### DISCUSSION TITLE
Payfast plugin for moodle - Logs required

### DISCUSSION THREAD

**Discussion by keens007** [community/community#77069] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I'm using the `Payfast` plugin for **moodle** which works well when integrated.

I ran into a issue where a payment was made via `PayFast`, but a user was not auto-enrolled into a course. 
Payment method was set on a course for enrolment once a payment is made.

If anyone can assist to find the logs in the moodle database that would be great since I don't see any activity for payments being made.
This plugin was installed on the site.
https://github.com/Payfast/mod-moodle

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @keens007 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
